src/libbson/doc/bson_init_from_json.rst

```diff
@@ -27,6 +27,11 @@ Description
 
 The ``bson_init_from_json()`` function will initialize a new :symbol:`bson_t` by parsing the JSON found in ``data``. Only a single JSON object may exist in ``data`` or an error will be set and false returned.
 
+Deprecated behavior: If there are multiple comma-separated JSONs in ``data``, the keys from all JSONs are merged in the returned BSON.
+For example, ``{"a": 1},{"b": 2}`` is parsed as ``{"a": 1, "b": 2}``. If the first character encountered after the last valid
+JSON object is ``{``, all following characters are ignored and no error is set.
+Otherwise, an error will be set and NULL returned. 
+
 ``data`` should be in `MongoDB Extended JSON <https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/>`_ format.
```

Comment:
```suggestion
``data`` should be in `MongoDB Extended JSON <https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/>`_ format.


Deprecated behavior
-------------------

If there are multiple comma-separated JSONs in ``data``, the keys from all JSONs are merged in the returned BSON.
For example, ``{"a": 1},{"b": 2}`` is parsed as ``{"a": 1, "b": 2}``.

If the first character encountered after the last valid
JSON object is ``{``, all following characters are ignored and no error is set.
Otherwise, an error will be set and NULL returned.

This deprecated behavior is subject to change in a future release.
```

Suggest splitting into a separate section and nothing this may change in a future release.